docker/validator/Dockerfile

```diff
@@ -26,6 +26,9 @@ FROM debian-base AS prod
 
 RUN apt-get update && apt-get install -y libssl1.1 && apt-get clean && rm -r /var/lib/apt/lists/*
 
+### Needed to run debugging tools like perf
+RUN apt-get update && apt-get install -y linux-tools-4.19 sudo procps
```

Comment:
n00b q: why do we need to `apt-get update` repeatedly?

Reply:
Actually we might not need it - but this seems to be the pattern here :D